mobile.js

```diff
@@ -49,3 +49,293 @@ closebtn.addEventListener('click', menuclose);
+if (removeimg !== null) modalimage.removeChild(removeimg);
+
+function popdesktop() {
+  const popupsection = document.querySelector('.popsection');
+  const popupwindow = document.querySelector('.popup_window');
+
+  popupsection.style.display = 'block';
+  popupwindow.style.display = 'block';
+  const popheading = document.querySelector('.update_heading');
+  popheading.textContent = 'Project-desktop name goes here';
+
+  const modalimage = document.querySelector('.modall-img');
+  const imgmodal = document.createElement('img');
+  modalimage.appendChild(imgmodal);
+  imgmodal.classList.add('modal-img');
+  imgmodal.src = 'images/Group 105.png';
+
+  const modalpara = document.querySelector('.modal-p');
+  const para = document.createElement('p');
+  modalpara.appendChild(para);
+  const para1 = document.createElement('p');
+  para.classList.add('para-desk');
+  para1.classList.add('para-desk1');
+  modalpara.appendChild(para1);
+  para.textContent = 'Lorem ipsum dolor sit amet,  aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut <br>labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis';
+  para1.textContent = 'Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut <br>labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi';
+
+  const divdesktop = document.querySelector('.divdestop');
+  const smallbtn = document.createElement('button');
+  const smallbtn1 = document.createElement('button');
+  divdesktop.appendChild(smallbtn);
+  divdesktop.appendChild(smallbtn1);
+  smallbtn.classList.add('arrow');
+  smallbtn1.classList.add('arrow1');
+  smallbtn.textContent = ' \u2190 Previous project';
+  smallbtn1.textContent = 'Next project  \u2192';
+
+  for (let i = 0; i < desktoptech.length; i += 1) {
+    const modallang = document.querySelector('.modal-lang ul');
+    const langli = document.createElement('li');
+    modallang.appendChild(langli);
+    const Lia = document.createElement('a');
+    langli.appendChild(Lia);
+    Lia.textContent = desktoptech[i];
+  }
+}
+
+btn1.addEventListener('click', popdesktop);
+btn2.addEventListener('click', popdesktop);
+
+const close2 = document.querySelector('.popup-close');
+// const modal-lang = document.querySelector('.modal-lang ul');
```

Comment:
- [ ] Please 🙏🏻 remove these line of code from your `file` because having commented code in the codebase makes it cluttered, confuses developers, and can create maintenance and version control issues. Kindly fix it and it should be avoided for clean, professional and standardized code.